service/HelloWorldService.ts

```diff
@@ -0,0 +1,11 @@
+const axios = require("axios")
+
+module.exports.getHelloWorld = async function() {
+    try { 
+        const response = await axios.get('http://localhost:8080/api/hello-world');
```

Comment:
Can you make `http://localhost:8080/` a configuration option. Currently if the URL is hard coded it won't work on AWS